orchagent/crmorch.h

```diff
@@ -63,6 +78,10 @@ class CrmOrch : public Orch
     void incCrmAclTableUsedCounter(CrmResourceType resource, sai_object_id_t tableId);
     // Decrement "used" counter for the per ACL table CRM resources (ACL entry/counter)
     void decCrmAclTableUsedCounter(CrmResourceType resource, sai_object_id_t tableId);
+    // Increment "used" counter for the per DASH ACL CRM resources (ACL group/rule)
+    void incCrmDashAclUsedCounter(CrmResourceType resource, sai_object_id_t groupId);
```

Comment:
Suggest using the same function naming convention for increment and decrement. Decrement has an additional Group within the name..

Reply:
Done